source/common/res/features/resize-inspector/main.js

```diff
@@ -28,6 +28,9 @@
 
                   ynabToolKit.shared.setToolkitStorageKey('budget-resize-inspector', asideWidth);
                   ynabToolKit.resizeInspector.asideWidth = asideWidth;
+
+                  let wdth = $('.budget-content').css('width');
```

Comment:
Would be good to name this variable `width`, but this will function, so I'll merge for now.

Reply:
Yeah. When the feature is refactored I'll look at renaming it.